qupath-gui-fx/src/main/java/qupath/lib/gui/charts/Charts.java

```diff
@@ -21,13 +21,7 @@
 
 package qupath.lib.gui.charts;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
I remain a bit attached to my import statements...

Reply:
Must be a case of very stubborn IDE settings, or me forgetting to change them on a different computer